Spigot-API-Patches/0193-final-Brand-support-PaperAPI.patch

```diff
@@ -0,0 +1,30 @@
+From a532a0c1f3438190efe6d95184026dfe79cae736 Mon Sep 17 00:00:00 2001
+From: DigitalRegent <misterwener@gmail.com>
+Date: Mon, 6 Apr 2020 19:53:29 +0200
+Subject: [PATCH] final Brand support (PaperAPI)
```

Comment:
I don't really like these commit messages, you don't need "final" and "(PaperAPI)" here as they're both implied.